cmd/protecodeExecuteScan.go

```diff
@@ -376,7 +376,7 @@ func uploadFile(utils protecodeUtils, config protecodeExecuteScanOptions, produc
 			combinedFileName = fmt.Sprintf("%v_%v", config.PullRequestName, fileName)
 		}
 
-		resultData := client.UploadScanFile(config.CleanupMode, config.Group, pathToFile, combinedFileName, version, productID, replaceBinary)
+		resultData := client.UploadScanFile(config.CleanupMode, config.Group, config.CustomDataJSONMap, pathToFile, combinedFileName, version, productID, replaceBinary)
```

Comment:
client.DeclareFetchURL() function should also accept customDataJSONMap, since fetch api also has a custom data header.